src/parser/jobs/mnk/modules/Brotherhood.tsx

```diff
@@ -137,59 +109,4 @@ export class Brotherhood extends Analyser {
 
 		return {...matchedWindow, status}
 	}
-
-	get expectedTargetCount(): number {
-		return this.partySize
-	}
-
-	/**
-	 * Note: This will not produce correct data until AFTER the OnComplete function of
-	 * the RiddleOfFire has occurred, as anytime before that the memoized map is still being
-	 * constructed
-	 */
-	get unmatchedWindows(): BrotherhoodWindow[] {
-		const foundEntries = Array.from(this.matchedWindows.values()).map(mw => mw.start)
-		return _.filter(this.windows, window => !foundEntries.includes(window.start))
-	}
-
-	private onApply(event: Events['statusApply']) {
-		if (this.currentWindow == null) {
-			this.currentWindow = {
-				start: event.timestamp,
-				targetsHit: 0,
-			}
-		}
-
-		this.currentWindow.targetsHit += 1
-	}
-
-	private onRemove(event: Events['statusRemove']) {
-		if (this.currentWindow == null) {
-			return
-		}
-
-		this.currentWindow.end = event.timestamp
-		this.windows.push(this.currentWindow)
-		this.currentWindow = undefined
-	}
-
-	private onComplete() {
-		const expectedBuffs = this.windows.length * this.partySize
-		const actualBuffs = _.sumBy(this.windows, 'targetsHit')
-
-		const windowsWithMissedBuffs = this.windows.filter(w => w.targetsHit !== this.partySize).length
-
-		this.suggestions.add(new TieredSuggestion({
-			icon: this.data.actions.BROTHERHOOD.icon,
-			content: <Trans id="mnk.brotherhood.suggestions.missed-players.content">
-				Try to make sure your <DataLink action="BROTHERHOOD"/> casts buff your full party with each use. Failing to do so is a raid damage loss.
-			</Trans>,
-			matcher: matchClosestHigher,
-			tiers: BROTHERHOOD_SEVERITY_TIERS,
-			value: (expectedBuffs - actualBuffs) / expectedBuffs,
-			why: <Trans id="mnk.brotherhood.suggestions.missed-players.why">
-				<Plural value={windowsWithMissedBuffs} one="# cast" other="# casts"/> of <DataLink action="BROTHERHOOD"/> missed hitting the entire party for a total of <Plural value={expectedBuffs - actualBuffs} one="# players" other="# players"/>.
-			</Trans>,
```

Comment:
This suggestion appears to be not working in the new approach, is that expected?  
![image](https://github.com/xivanalysis/xivanalysis/assets/26804378/87cbe519-cdfc-4971-beba-b9ed07895699)

https://xivanalysis.com/fflogs/wkGCrNh2p3BWVtzx/2/63 generates this suggestion in production, but not in the PR.

Reply:
We made the decision for basically every other raid buff window to not ding the player for missing someone since that's not fully under their control (party member in Narnia or dead for example). This just brings MNK in line with other jobs, and is intended.